src/components/internal/OverlayTrigger.tsx

```diff
@@ -80,7 +80,8 @@ export function OverlayTrigger(props: OverlayTriggerProps) {
   );
 
   return (
-    <div css={Css.relative.dib.$}>
+    // Add `line-height: 0` to prevent the Icon button and Avatar buttons from inheriting the line-height, causing them to be taller than they should.
+    <div css={Css.dib.add("lineHeight", 0).$}>
```

Comment:
Nit: We don't have `lh0` but we do have `lh(0)` for next time around (no need to do a follow up). Might be cute to add `lh0` / `lh1` for increments :shrug: .

Also great in-source comment!